cpp/arcticdb/processing/clause.cpp

```diff
@@ -304,6 +304,19 @@ Composite<ProcessingUnit> AggregationClause::process(std::shared_ptr<Store> stor
                                                 // 11.01 seconds with caching
                                                 // Not worth worrying about right now
                                                 robin_hood::unordered_flat_map<RawType, size_t> offset_to_group;
+
+                                                bool is_sparse = col.column_->is_sparse();
+                                                using optional_iter_type = std::optional<decltype(input_data.bit_vector()->first())>;
+                                                optional_iter_type iter = std::nullopt;
+                                                size_t previous_value_index = 0;
```

Comment:
Nope, `previous_value_index` is the index right after the last value set. Depending on the current iterator, it can be the index of a missing value, or of a value to set. I'm gonna change the logic so that it's always the index of the last value set.